torchani/units.py

```diff
@@ -0,0 +1,124 @@
+r"""Unit conversion factors
+
+All values derived from:
```

Comment:
Could you also add this file to `docs/api.rst`?

Reply:
Done!